rebekka-source/Rebekka/StreamOperation.swift

```diff
@@ -70,6 +70,7 @@ internal class StreamOperation: Operation, NSStreamDelegate {
     }
     
     func finishOperation() {
+        NSThread.sleepForTimeInterval(0.1)
```

Comment:
Hm, I think I have the same problem. Can't reproduce it myself too. Does it look like this one?
```
1	CFNetwork	NetConnection::dequeue(void*) + 40
2	CFNetwork	CoreStreamBase::_streamInterface_Close() + 92
3	CoreFoundation	_CFStreamClose + 104
```